src/main/java/au/com/sealink/quicktravel/client/models/Booking.java

```diff
@@ -26,4 +74,93 @@ public String getState() {
     public void setState(String state) {
         this.state = state;
     }
+
+    public Client getClient() {
+        return client;
+    }
+
+    public void setClient(Client client) {
+        this.client = client;
+    }
+
+    public String getReference() {
+        return reference;
+    }
+
+    public void setReference(String reference) {
+        this.reference = reference;
+    }
+
+    public Date getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(Date createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    public Date getUpdatedAt() {
+        return updatedAt;
+    }
+
+    public void setUpdatedAt(Date updatedAt) {
+        this.updatedAt = updatedAt;
+    }
+
+    public int getCountryId() {
+        return countryId;
+    }
+
+    public void setCountryId(int countryId) {
+        this.countryId = countryId;
+    }
+
+    public String getPublicComments() {
+        return publicComments;
+    }
+
+    public void setPublicComments(String publicComments) {
+        this.publicComments = publicComments;
+    }
+
+    public String getInternalComments() {
+        return internalComments;
+    }
+
+    public void setInternalComments(String internalComments) {
+        this.internalComments = internalComments;
+    }
+
+    public String getPostCode() {
+        return postCode;
+    }
+
+    public void setPostCode(String postCode) {
+        this.postCode = postCode;
+    }
+
+    public String getExternalIdentifier() {
+        return externalIdentifier;
+    }
+
+    public void setExternalIdentifier(String externalIdentifier) {
+        this.externalIdentifier = externalIdentifier;
+    }
+
+    public int getGrossInCents() {
+        return grossInCents;
+    }
+
+    public void setGrossInCents(int grossInCents) {
+        this.grossInCents = grossInCents;
+    }
+
+    public int getNettInCents() {
+        return nettInCents;
+    }
+
+    public void setNettInCents(int netInCents) {
```

Comment:
Minor but you missed the variables inside this method

Reply:
Well caught (by Sean's "net")